### PR TITLE
Adds first draft of freelance agreement 

### PR DESCRIPTION
**Pull request by iteles**
This still feels too long for me and I think it's maybe because my natural ['voice'](http://copyhackers.com/2013/01/copywriting-tone-how-to/) is a little lengthy. Now we need to start _deleting stuff_ from this draft. 

@nelsonic This is _very much_ a first draft. Could you please **ruthlessly review** and add a commit to this branch? We can use the branch to track our revision process and document the morphing of the agreement.

Would :heart: to have more people's opinions on this! #27

### CODE REVIEW

**Comment by iteles** [comment]
_NB. With inspiration from:_ 
http://www.apache.org/licenses/icla.txt & https://cla.developers.google.com/about/google-individual

File: freelance-agreement.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+
+## Tracking Work
+
+You acknowledge and agree that:
+
+1. **Work is tracked _exclusively_ on GitHub issues**.
++ Each task should have its own GitHub issue and be clearly related to a _component_ of the **dwyl app**;
++ If you discuss something related to a story/feature/task _verbally_, always ensure it is captured in a comment in the GitHub issue for posterity; your memory is not as good as you think it is and the person reviewing the feature/code might not be in the same country as you! As much detail/context as possible in the issue is always better.
++ Tasks _**must**_ **be _estimated_** (_in minutes - not "points" - by everyone present and an **average recorded** as the estimate_) **_before_ being _started_**.
```
Comment (iteles) [inline freelance-agreement.md]: "everyone present" won't work in the context of remote freelancers. "by at least 3 other dwyl contributors"?

File: freelance-agreement.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+1. The contributions you make to the codebase are **_your original work_**
+(_never copied from somewhere else_)
+2. By committing your code into a dwyl-owned repository, **you grant worldwide, and irrevocable but _non-exclusive_ copyright with no further consideration**
+3. This is an open source project, so in addition to **dwyl ltd**. using your (and all of our) contributions, the codebase may also be used by other people (3rd parties) without further consent or consideration.
++ Your contribution **does not grant you a _claim_ to any earnings** (past, present or future) of **dwyl Ltd**. or any other individuals or companies that use your code.
++ You **do not have any _obligation_ to provide support for your code**, however you are encouraged to remain a contributor/supporter for as long you find it personally beneficial/rewarding;
++ If you use the original works you contribute to the dwyl codebase in future projects we ask that you provide **attribution** back to the project where you originally made the contribution.
+
+We (_dwyl Ltd_.) agree to the following:
```
Comment (iteles) [inline freelance-agreement.md]: I would favour using 'dwyl' and 'dwyl ltd.' interchangeably (defining this in the terms/definitions) as 'dwyl ltd' is very harsh and reduces readability. It's a long document and I would prefer it to 'flow'. @nelsonic Do you have strong objections to this?

**Comment by iteles** [comment]
@nelsonic those comments on outdated diffs above are still relevant questions!

**Comment by nelsonic** [comment]
@iteles I'm a bit lost. do you want me to merge this now?

**Comment by iteles** [comment]
@nelsonic - no, I'd like you to read the 'comments on the outdated diffs' that appear directly above my last commit (you'll have to expand them first) and respond to them please.

**Comment by iteles** [comment]
Moved _tracking work_ and _definition of done_ out of the main freelance agreement into a separate document for:
a) Ease of updating (hooray for version control)
b) Keeping the agreement itself as stable as possible, therefore moving sections most likely to change out of it.

**Comment by nelsonic** [comment]
Agreed.

**Pull request merged**